website/profile/views.py

```diff
@@ -102,6 +104,8 @@ def resend_confirmation(auth):
     data = request.get_json()
 
     validate_user(data, user)
+    if not throttle_period_expired(user.email_last_sent, settings.SEND_EMAIL_THROTTLE):
+        raise HTTPError(httplib.BAD_REQUEST, data={'message_long': 'Please wait 30 seconds before sending another confirmation email.'})
```

Comment:
Ah, this is in the view, right. So maybe, 'Too many requests. Please wait a while before sending another confirmation email.'